### DISCUSSION TITLE
Gif isn't included in website despite gh-pages branch having it

### DISCUSSION THREAD

**Discussion by Andre601** [mmistakes/minimal-mistakes#3010] [Q&A] [answered]
I encountered a strange issue where a gif that I added under `/assets/images` isn't included on the final page.
The `gh-pages` branch does have the file but for yet unknown reasons is the file not included on the website itself. The Source tab in the dev tools only shows the PNGs but the Gif isn't there at all.

Is this a Jekyll issue or does this theme for whatever reason not like/support gifs?

**Comment by mmistakes** [comment, marked as answer]
Not a theme issue.

**Reply by Andre601** [reply]
Yeah. Just noticed that the url was wrong.
Was just weird that the image didn't show in the source tab...

**Reply by mmistakes** [reply]
It’s not going to show in the source tab if the browser can’t find it due to having the wrong path in the HTML source.
